acls_types.go

```diff
@@ -125,7 +125,7 @@ func (autoApprovers *AutoApprovers) GetRouteApprovers(
 			return nil, err
 		}
 
-		if autoApprovedPrefix.Bits() >= prefix.Bits() &&
+		if prefix.Bits() >= autoApprovedPrefix.Bits() &&
```

Comment:
Is this check even necessary, given the `Contains` check below?

Reply:
Never mind I see, `Contains` operates on a single IP, hence the need for this check.

Reply:
probably not honestly, I can't recall my thinking at the time but it might've been to bail out quicker if the enabledRoute was already strictly larger than the autoApprove entry being assessed. Might've also been me just being paranoid...

Reply:
ah yeah that makes sense, I'd forgotten entirely and it's pretty late here 😅